CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ This CHANGELOG follows the format listed [here](https://github.com/sensu-plugins
 
 ## [Unreleased]
 
+## [10.0.1] - 2017-12-06
```

Comment:
we do all the versioning on the maintainer end and have things be accepted into [Unreleased Changes](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md#unreleased-change) there are several reasons for this:
- there is no guarantee on merge order
- there is no SLA on review times, the dates in the changelogs coincide when the release was made not when the work for the work involved in the release.
- in this case `10.0.3` has been released
- per our [versioning guidelines](https://github.com/sensu-plugins/community/blob/master/HOW_WE_VERSION.md) which is essentially [semver](https://semver.org/spec/v2.0.0.html) this would be a new feature which would consitute a minor bump rather than a patch.

Just kill line 8 and leave the rest which looks good, great job on a good changelog entry.